Client/game_sa/CModelInfoSA.cpp

```diff
@@ -34,6 +34,7 @@ std::unordered_map<std::uint32_t, std::map<eVehicleDummies, CVector>> CModelInfo
 std::map<CTimeInfoSAInterface*, CTimeInfoSAInterface*>                CModelInfoSA::ms_ModelDefaultModelTimeInfo;
 std::unordered_map<DWORD, unsigned short>                             CModelInfoSA::ms_OriginalObjectPropertiesGroups;
 std::unordered_map<DWORD, std::pair<float, float>>                    CModelInfoSA::ms_VehicleModelDefaultWheelSizes;
+std::map<unsigned short, int>                                         CModelInfoSA::ms_DefaultTxdIDMap;
```

Comment:
You should consider using a fixed-size array, it's faster, and uses less memory.

Reply:
Fixed-size array will use at least 320 KB memory

Reply:
Memory fragmentation >>> usage

Reply:
This struct will be empty on 95% servers. I see no reason to optimize this struct with memory cost for now.